.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
     - sudo apt-get install build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
 install:
     - npm config set prefer-offline false
-    - npm install -g enactjs/cli#develop
+    - npm install -g @enact/cli@5.1.1
```

Comment:
I guess we don't need this change anymore.

Reply:
Or should we do like
- npm install -g @enact/cli
as we changed from develop branch?

Reply:
I changed as develop branch.